public/main.js

```diff
@@ -32,5 +40,19 @@ function createWindow() {
   mainWindow.on('closed', () => {
     mainWindow = null;
   });
+
+  const poster = io.on('connection', (socket) => {
+    console.log('a user connected');
+
+    expressApp.post('/posturl', (request, response) => {
+      socket.emit('post_received', request.body);
+      response.status(200);
```

Comment:
Let‘s add some basic error handling here. Thinking we should reject any non-JSON or non-XML data